.NET/Microsoft.Recognizers.Text.Number/Parsers/IIndianNumberParserConfiguration.cs

```diff
@@ -0,0 +1,180 @@
+﻿using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.Globalization;
+using System.Linq;
+using System.Text.RegularExpressions;
+
+namespace Microsoft.Recognizers.Text.Number
+{
+    public interface IIndianNumberParserConfiguration : INumberParserConfiguration
+    {
+        ImmutableDictionary<string, double> DecimalUnitsMap { get; }
```

Comment:
Can you add comments on this interface to describe why these extra properties are needed over INumberParserConfiguration. This may be helpful for future developers.